functions.gallery.php

```diff
@@ -8,7 +8,16 @@ function __construct() {
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
 	}
 
-	function admin_init() {
+	function admin_init() 
+	{
+		/**
+		 * Filter the available gallery types.
+		 *
+		 * @since 2.5.1
+		 *
+		 * @param array  $instance contains array of the default thumbnail grid gallery type.
```

Comment:
Could you change `$instance` into the actual value in the filter? You'll need to do it for both filters.